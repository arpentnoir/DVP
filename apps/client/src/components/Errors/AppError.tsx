import { Text } from '@dvp/vc-ui';
import { Stack } from '@mui/material';

export const AppError = () => {
  return (
    <Stack
      sx={{
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center',
        display: 'flex',
      }}
    >
      <Stack spacing={2}>
        <Text variant="h6" fontWeight="bold">
          500
        </Text>
        <Text variant="h4">Something went wrong</Text>
        <Text>Please contact us for further information.</Text>
      </Stack>
    </Stack>
  );
};
