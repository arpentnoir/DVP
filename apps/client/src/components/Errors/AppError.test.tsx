import { render } from '@testing-library/react';
import { AppError } from './AppError';

describe('AppError', () => {
  it('should render correctly', () => {
    const { baseElement } = render(<AppError />);

    expect(baseElement).toMatchSnapshot();
  });
});
