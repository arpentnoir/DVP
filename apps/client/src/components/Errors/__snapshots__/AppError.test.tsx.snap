// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppError should render correctly 1`] = `
<body>
  <div>
    <div
      class="css-1ootpi0-MuiStack-root"
    >
      <div
        class="css-1p5q5e5-MuiStack-root"
      >
        <h6
          class="MuiTypography-root MuiTypography-h6 css-tjvwag-MuiTypography-root"
          tabindex="0"
        >
          500
        </h6>
        <h4
          class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
          tabindex="0"
        >
          Something went wrong
        </h4>
        <p
          class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          tabindex="0"
        >
          Please contact us for further information.
        </p>
      </div>
    </div>
  </div>
</body>
`;
